### PR TITLE
Rwd/sidebar and council

### PR DESCRIPTION
**Pull request by eshark9312**
- #1794

Designs:
- [Council](https://www.figma.com/file/EaObQEN95O0IWjSVTUQ5NN/Pioneer-RWD?type=design&node-id=5%3A6465&mode=design&t=DZjuVGp9Whk2e2D9-1)
- [Navigation](https://www.figma.com/file/EaObQEN95O0IWjSVTUQ5NN/Pioneer-RWD?type=design&node-id=706-202434&mode=design&t=mn7hq2HS5K3k3isW-4)
- [Navigation prototype](https://www.figma.com/proto/EaObQEN95O0IWjSVTUQ5NN/Pioneer-RWD?page-id=703%3A21201&type=design&node-id=733-110109&viewport=12399%2C-184%2C2.03&t=VYX0t4dq7BvnO4Jj-1&scaling=min-zoom&starting-point-node-id=733%3A110109)

### CODE REVIEW

**Review by thesan — CHANGES_REQUESTED** [review]
It's fine if you have already addressed this in further commits, but in case you haven't the past council pages (not the list but the page of a given past council) is not ready yet:

Header and statistics:
image

The "Go To Proposal" buttons (I think it's the `PastCouncilProposalsItem` component)
image

File: packages/ui/src/app/components/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        >
+          <MenuIcon />
+        </MenuIconWrapper>
+      </Wrapper>
+      {isMobile && open && <SideBarSlider setOpen={setOpen} />}
+    </>
+  )
+}
+
+const Wrapper = styled.div`
```
Comment (thesan) [inline packages/ui/src/app/components/NavBar.tsx:33]: [The design](https://www.figma.com/file/EaObQEN95O0IWjSVTUQ5NN/Pioneer-RWD?type=design&node-id=706-202460&mode=dev) has a `box-shadow: 0px -1px 0px 0px rgba(187, 217, 246, 0.13) inset;` it's really subtle but I think it makes scrolling through white area better

File: packages/ui/src/common/components/page/Sidebar/Navigation.tsx [hunk 2]
```diff
@@ -3,15 +3,19 @@ import styled from 'styled-components'
...
   max-width: 226px;
   height: 100vh;
   z-index: ${ZIndex.navbar};
+
+  @media (min-width: 1024px) {
+    display: flex;
+  }
 `
 
 export const NavigationInnerWrapper = styled.div`
```
Comment (thesan) [inline packages/ui/src/common/components/page/Sidebar/Navigation.tsx:21]: Finally please reduce the height of the `barheader` row to `72px` on mobile to fully match [the sliding navigation design](https://www.figma.com/file/EaObQEN95O0IWjSVTUQ5NN/Pioneer-RWD?type=design&node-id=708-202748&mode=dev)

**Review by thesan — CHANGES_REQUESTED** [review]
Great work! Just 3 small more comments. Also the member side panes are not usable yet but I think this should be addressed in some future PR.

**Review by thesan — APPROVED** [review]
Once again great work :raised_hands: 

I just created a new branch called `responsive` and I re-targeted this PR to it. Once this PR is merged please pull this new branch on the changes you will create the next PR from (otherwise you can rebase part of your changes on it if you prefer).

**Comment by ivanturlakov** [comment]
✅ Tested on https://dao-git-responsive-joystream.vercel.app/#/council
Mainnet

Screenshot 2023-10-26 at 20 43 36
Screenshot 2023-10-26 at 20 44 27
Screenshot 2023-10-26 at 20 45 36

**Comment by eshark9312** [comment]
@ivanturlakov 
I addressed this problem on the last [RWD PR](https://github.com/Joystream/pioneer/pull/4607), you can check [here](https://dao-38ivwh7by-joystream.vercel.app/).

**Pull request merged**